fdbserver/TagThrottler.actor.cpp

```diff
@@ -175,6 +175,10 @@ class TagThrottlerImpl {
 	PrioritizedTransactionTagMap<ClientTagThrottleLimits> getClientRates() {
 		return throttledTags.getClientRates(autoThrottlingEnabled);
 	}
+	PrioritizedTransactionTagMap<double> getRates() const {
+		// TODO: Implement (currently not supported)
+		return {};
+	}
```

Comment:
If not implemented, we'd better throw an exception here in case someone uses this TagThrottler but enable Proxy side throttling.